test/unit/helpers/xhr/MFA_ENROLL_ACTIVATE_email.js

```diff
@@ -0,0 +1,69 @@
+define({
+  "status": 200,
+  "responseType": "json",
+  "response": {
```

Comment:
not for this PR, but maybe something to think about: could we import `playground/mocks/api/v1/authn/data/enroll-activate-email.json` here instead of writing the json response again? Would that be a good idea or are there some side effects?

Reply:
Well, it's good idea and I had thought about it but didn't find time to do some investigation because I'm not sure
- whether I could import the JSON directly via AMD module syntax? It used to need some static loader like `json!/path/to/file`. 
- Or would webpack json loader plugin be helpful?
- otherwise, we may have to convert AMD to ES6 import style.